sourcemash/api/feeds.py

```diff
@@ -18,14 +20,75 @@
 
 class FeedListAPI(Resource):
```

Comment:
Would you want to add it now? I'd only encourage it because we're making enormous API changes here so it makes sense as part of this PR more than as a future issue.